client/blocks/signup-form/passwordless.jsx

```diff
@@ -307,7 +329,16 @@ class PasswordlessSignupForm extends Component {
 			<div className="signup-form__passwordless-form-wrapper">
 				<LoggedOutForm onSubmit={ this.onFormSubmit } noValidate>
 					<ValidationFieldset errorMessages={ errorMessages }>
-						<FormLabel htmlFor="signup-email">{ this.getLabelText() }</FormLabel>
```

Comment:
Why remove the `FormLabel` component? All the other `<Form...` components are here, let's keep this congruent. Also, the function says get Label **Text**, so I'd expect for it to return text 👍